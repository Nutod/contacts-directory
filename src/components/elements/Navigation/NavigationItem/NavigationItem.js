import React from "react";
import styled from "styled-components";
import theme from "../../Icons/Colors";

const ListItem = styled.li`
	list-style: none;
	margin-right: 2.5rem;
	padding: 1rem;
	cursor: pointer;
	transition: all 0.2s;
	font-size: 1.3rem;
	color: #aaaaaa;

	&:hover {
		color: ${theme.purple};
	}

	@media (max-width: 790px) {
		padding: 0;
	}
`;

// This component will implement links

export default function NavigationItem({ children }) {
	return <ListItem>{children}</ListItem>;
}
