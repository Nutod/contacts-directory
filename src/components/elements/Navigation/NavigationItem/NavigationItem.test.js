import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import toJson from "enzyme-to-json";
import NavigationItem from "./NavigationItem";

Enzyme.configure({
	adapter: new Adapter()
});

describe("<NavigationItem />", () => {
	const wrapper = shallow(<NavigationItem />);

	it("Should match snapshot", () => {
		expect(toJson(wrapper)).toMatchSnapshot();
	});
});
