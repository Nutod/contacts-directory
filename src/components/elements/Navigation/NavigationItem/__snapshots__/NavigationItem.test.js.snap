// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NavigationItem /> Should match snapshot 1`] = `<styled.li />`;
